api/v1alpha1/instancegroup_types.go

```diff
@@ -372,6 +373,12 @@ func (c *EKSConfiguration) GetSubnets() []string {
 	}
 	return c.Subnets
 }
+func (c *EKSConfiguration) GetSuspendProcesses() []string {
+	if c.SuspendedProcesses == nil {
```

Comment:
I don't think this can be nil, just an empty slice

Reply:
Hi @eytan-avisror ,
While debugging, I see that it is nil and has hence reached line no. 378

Here's a snapshot of its value when the input yaml contains no mention of the "suspendProcesses" field

<img width="1350" alt="Screen Shot 2020-06-03 at 11 41 46 AM" src="https://user-images.githubusercontent.com/9319820/83602122-965ff980-a58f-11ea-88cf-ff730dc99ea6.png">

Please do let me know if I should leave this piece as is or whether you would like me to perform any modifications to it.

Thank you.

Reply:
Hi @eytan-avisror ,
Would you prefer the 
`if len(c.SuspendedProcesses) == 0:` instead of `c.SuspendedProcesses == nil` ? 

I see that checking length of a nil slice doesn't result in an error in golang (https://tour.golang.org/moretypes/12) 

Please let me know which one you prefer.

Thank you